tests/backends/oracle/tests.py

```diff
@@ -103,15 +103,17 @@ def test_hidden_no_data_found_exception(self):
                 cursor.execute('DROP TRIGGER "TRG_NO_DATA_FOUND"')
 
     def test_password_with_at_sign(self):
+        from django.db.backends.oracle.base import Database
+
```

Comment:
Other than my question (to better understand the fix), LGTM :+1:

Reply:
We need to catch a different error, because we're calling a different method that doesn't wrap database-specific errors with Django's `DatabaseError`.